main/templates/base/fetc_form_base.html

```diff
@@ -3,8 +3,10 @@
 {% block content %}
     <div class="uu-container">
         {# A block one can use to insert content before the stepper/form container #}
-        {% block pre-form-container-content %}{% endblock %}
+        <div class="col-12 text-center">
+            {% block pre-form-container-content %}{% endblock %}
 
+        </div>
```

Comment:
I'm a bit unsure about this one. It does solve the problem you probably encountered, but it does introduce some limitations. 

In my intended usecase you would actually add this `col-12` div as part of the content you add in this content block. 

That way, you can choose to use a different col-layout (like a `col-3` and a `col-9` to mimick the form-layout below). 

In any case, the `text-center` class doesn't make sense here. Why would you want to force all content inside to have centered text? That's something you should decide per-usage